### PR TITLE
Refactoring & clean-up [sc-719]

### PR DESCRIPTION
**Pull request by MDobak**

### CODE REVIEW

File: pkg/price/provider/graph/async_provider.go [hunk 1]
```diff
@@ -20,44 +20,45 @@ import (
...
-	"github.com/chronicleprotocol/oracle-suite/pkg/gofer/graph/feeder"
-	"github.com/chronicleprotocol/oracle-suite/pkg/gofer/graph/nodes"
+	"github.com/chronicleprotocol/oracle-suite/pkg/price/provider"
+	"github.com/chronicleprotocol/oracle-suite/pkg/price/provider/graph/feeder"
+	"github.com/chronicleprotocol/oracle-suite/pkg/price/provider/graph/nodes"
+
 	"github.com/chronicleprotocol/oracle-suite/pkg/log"
 )
 
 const LoggerTag = "ASYNC_GOFER"
```
Comment (jar-o) [inline pkg/price/provider/graph/async_provider.go:30]: since this has been renamed, should this be `ASYNC_GRAPH` now?

File: cmd/gofer/cmd.go [hunk 2]
```diff
@@ -27,7 +27,7 @@ func NewRootCommand(opts *options) *cobra.Command {
 		Version: opts.Version,
 		Short:   "Tool for providing reliable data in the blockchain ecosystem",
 		Long: `
-Gofer is a CLI interface for the Gofer Go Library.
+Graph is a CLI interface for the Graph Go Library.
```
Comment (jar-o) [inline cmd/gofer/cmd.go:30]: a little confused by this description. `gofer` is still the CLI correct? should this instead read `Gofer is a CLI interface for the Graph Go library`

Comment (MDobak) [reply cmd/gofer/cmd.go:30]: I changed it by mistake, good catch!

**Review by jar-o — APPROVED** [review]
lgtm. a lotta renames heh

**Comment by MDobak** [comment]
> lgtm. a lotta renames heh

@jar-o We discussed a lot about removing product names from the code, so I did it in this PR. Also, I'm terrible at naming things, but I hope these new names make more sense.

**Review by teghnet — APPROVED** [review]

**Comment by teghnet** [comment]
I like the generic utils

**Pull request merged**